daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -31,33 +31,43 @@ import Test.Main
 import Test.Tasty
 import Test.Tasty.HUnit
 import qualified Web.JWT as JWT
+import qualified Data.ByteString.Lazy as BSL
+import qualified Data.HashMap.Strict as HMS
+import Data.Aeson
 
+import DA.Directory
 import DA.Bazel.Runfiles
 import DA.Daml.Helper.Run
 import SdkVersion
 
 main :: IO ()
-main =
-    withTempDir $ \tmpDir -> do
+main = do
+    setEnv "TASTY_NUM_THREADS" "1" True
```

Comment:
Why did you move this above the explaining comment?

Reply:
Actually, I introduced the explaining comment 😄 I'll move the comment above the action.